lib/wayforpay/payments.rb

```diff
@@ -44,5 +44,33 @@ def self.get_client(attrs = {})
       request_params = Constants.get_client_params.merge(attrs)
       Wayforpay::Request.(Constants::GET_CLIENT_ENCRYPT_FIELDS, request_params)
     end
+
+    # required attrs: orderReference, orderDate, amount, currency, productName[], productCount[], productPrice[]
+    def self.purchase_form(attrs = {})
```

Comment:
Since it is UI thing It should be as a rails view helper helper. We use this class to process API requests

Reply:
Any suggestions on where to put it? Would `lib/helpers.rb` be a good place to put these methods there?

Reply:
`lib/wayfopay/helpers.rb`